src/main/java/com/chaos131/auto/AutoBuilder.java

```diff
@@ -28,6 +28,9 @@ public class AutoBuilder {
     private Map<String, Function<ParsedCommand, Command>> m_knownCommands = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
     private Map<String, Function<String, IAutoCondition>> m_knownConditions = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
 
+    /**
+     * Standard Constructor
+     */
     public AutoBuilder() {
```

Comment:
If we stick to PathPlanner (I don't see why we wouldn't), I think we can gut the entire auto folder

Reply:
I wasn't sure what to do about this, or the LogManager stuff. I could imagine them becoming useful down the line, so I didn't want to just delete stuff. I just don't know where or how that would manifest.

It would be a lot less javadocs to write if we tossed the folder though!